commit 7e683318c48498500682fc3650469d1a9865904f
Author: Michael Gernoth <michael@gernoth.net>
Date:   Tue May 20 16:46:48 2014 +0200

    omap4-common: moto_audio_wrapper: fix set_mode for ICS blob
    
    Change-Id: Iee07478f9121dc0ca17f00c78fd95fb6a947fa93

diff --git a/audio/wrapper.c b/audio/wrapper.c
index 79d4fa4..303e1fe 100644
--- a/audio/wrapper.c
+++ b/audio/wrapper.c
@@ -432,7 +432,7 @@ static int wrapper_set_mode_ics(struct audio_hw_device *dev, audio_mode_t mode)
 {
     ALOGI("ICS: set_mode: %d", mode);
 
-    ics_hw_dev->set_voice_volume(ics_hw_dev, mode);
+    ics_hw_dev->set_mode(ics_hw_dev, mode);
 
     return wrapper_set_mode(dev, mode);
 }
